test/tic_tac_toe_test.exs

```diff
@@ -2,7 +2,32 @@ defmodule TicTacToeTest do
   use ExUnit.Case
   import ExUnit.CaptureIO
 
-  test "displays the welcome message" do
-    assert capture_io(fn -> TicTacToe.welcome_message() end) === "Welcome to Tic-Tac-Toe"
+
+  describe "start/1" do
+    test "displays welcome message and board" do
+      assert capture_io(fn -> TicTacToe.start(CommandLine) end) ===
```

Comment:
I feel like this test belongs to `CommandLine`. I assume `CommandLine` is a wrapper around `IO` and is the only thing that directly interacts with it so it would make sense to actually test the integration by using `capture_io`. In this case `TicTacToe` doesn't have any knowledge of IO because it's delegating to `CommandLine`.

Reply:
So I should put this in the command_line_test file instead? This mostly makes sense to me, I could just use a little clarification on the reasoning. The only thing that confuses me about this test transfer is TicTacToe.start() is being called here, so that makes me think that it should reside in the tic_tac_toe test. From what I understand, you are saying the test is not really testing anything about the TicTacToe module and instead the integration test is testing the interaction between my `CommandLine` module (Which TicTacToe is delegating all I/O responsibility to) and the actual Terminal/CommandLine (via capture_io) so the test actually belongs in command_line_test? Now that I'm writing this out it makes a lot of sense, I appreciate all of the feedback and answering any questions I have on it 🙏  Thank you!

Reply:
I just remember @elijahsullivan had mentioned in the Slack that this is an integration test which makes sense. Ignore my comments above. My bad!

Reply:
Initially I was a bit confused looking at both tests and completely forgot about @elijahsullivan 's slack message which explains what this test is. I'm fine with this test being here as-is. I do wonder if we can leave a comment for this test to clarify that intent and also act as a reminder that we should move it to another spot in the future? or just adding `(integration test)` to the description? Thoughts @elijahsullivan @jonmanahan @matteeyao?